File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    li = list(s.split())
    an = []
    answer = []
    for i in li:
        an.append(int(i))
    an.sort()
    answer.append(str(an[0]))
    answer.append(str(an[-1]))
    return ' '.join(answer)

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] strArr = s.split(" ");
        
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        
        for(String str : strArr) {
            int i = Integer.valueOf(str);
            if(max < i) max = i;
            if(min > i) min = i;
        }
        
        return min + " " + max;
    }
}